### PR TITLE
lab-aaron

### PR DESCRIPTION
**Pull request by arn1313**

### CODE REVIEW

File: lab-aaron/src/component/category-form/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  handleSubmit(e) {
+    e.preventDefault();
+    this.props.onComplete(Object.assign({}, this.state));
+    this.setState({title: ''});
+  }
+
+  render() {
+    return (
+      <form className="category-form" onSubmit={this.handleSubmit}>
```
Comment (izzybaer) [inline lab-aaron/src/component/category-form/index.js:33]: something inside of your form is off, when I type in an expense name and then go to type in a price the price appears in both the name input box and the price input box.

Comment (arn1313) [reply lab-aaron/src/component/category-form/index.js:33]: Yep, been trying to get some time to go figure it out and complete 27/28.
Been sick has been kicking my butt, but im planning on working through the weekend/weekends? until we are done to get them fixed.

File: lab-aaron/src/component/category-form/index.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        <input
+          required
+          type="text"
+          name="name"
```
Comment (izzybaer) [inline lab-aaron/src/component/category-form/index.js:37]: whatever is inside of name="name" (in this case, you've called it name) needs to match the value exactly, which needs to be a prop on the state of that component, so 
```
name="title"
value={this.state.title}
```

Comment (arn1313) [reply lab-aaron/src/component/category-form/index.js:37]: Awesome Thanks!

File: lab-aaron/src/component/category-form/index.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+
+class CategoryForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```
Comment (izzybaer) [inline lab-aaron/src/component/category-form/index.js:7]: this needs a title, id, timestamp, and budget as props.

File: lab-aaron/src/component/dashboard-container/index.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {categoryCreate} from '../../action/category-actions';
+import CategoryForm from '../category-form';
+import CategoryItem from '../category-item';
+
+class DashBoardContainer extends React.Component {
+  componentDidMount() {
+    console.log('__DASHBOARD__'. this);
+    // this.props.categoryCreate({title: 'Create a Category'});
```
Comment (izzybaer) [inline lab-aaron/src/component/dashboard-container/index.js:10]: corpse code

**Pull request merged**